pkg/validators/ocm_client.go

```diff
@@ -0,0 +1,39 @@
+package validators
+
+import (
+	"context"
+
+	amv1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+)
+
+// OCMError abstracts behavior required for validators to identify the underlying
+// causes of OCM related errors.
+type OCMError interface {
+	// ServerSide returns 'true' if an instance of OCMError was caused
+	// by a server-side issue.
+	ServerSide() bool
+}
+
+// IsOCMServerSideError determines if the given error is both an instance of OCMError
+// and was caused by a server-side issue.
+func IsOCMServerSideError(err error) bool {
+	ocmErr, ok := err.(OCMError)
+
+	return ok && ocmErr.ServerSide()
+}
+
+// OCMClient abstracts behavior required for validators which request data
+// from OCM to be implemented by OCM API clients.
+type OCMClient interface {
+	SKURuleGetter
```

Comment:
Why the extra interface separation? What else will implement the SKURuleGetter besides OCMClient?